the-blue-alliance-ios/View Controllers/MyTBA/MyTBAViewController.swift

```diff
@@ -148,15 +149,10 @@ class MyTBAViewController: ContainerViewController, GIDSignInUIDelegate {
         deleteSubscriptions.resultType = .resultTypeObjectIDs
 
         for (viewController, deleteRequest) in zip([favoritesViewController, subscriptionsViewController], [deleteFavorites, deleteSubscriptions]) {
-            do {
-                let result = try self.persistentContainer.persistentStoreCoordinator.execute(deleteRequest, with: persistentContainer.viewContext) as? NSBatchDeleteResult
-                let objectIDArray = result?.result as? [NSManagedObjectID] ?? []
-                let changes = [NSDeletedObjectsKey: objectIDArray]
-                NSManagedObjectContext.mergeChanges(fromRemoteContextSave: changes, into: [persistentContainer.viewContext])
-            } catch {
-                // TODO: Mark these for deletion later...
-                Crashlytics.sharedInstance().recordError(error)
-            }
+            let result = try! self.persistentContainer.persistentStoreCoordinator.execute(deleteRequest, with: persistentContainer.viewContext) as? NSBatchDeleteResult
```

Comment:
These changes didn't make it in to https://github.com/the-blue-alliance/the-blue-alliance-ios/commit/5d7f1ba05734d067b054790c1f853a301dbec11e